articles/microsites/add-login/add-login-regular-web-app.md

```diff
@@ -33,23 +33,13 @@ For security in server-side web apps, Auth0 uses the [Regular Web App Login Flow
 ## Implementation overview
 
 ::: steps
-  1. <strong>Configure the sign-in methods</strong><br/><br/>Auth0 supports a wide range of authentication methods: regular username/password (users can be stored in Auth0 or your own database), social (i.e., Google, Facebook, and 50+ other providers), passwordless (email magic link, email code, and phone code), and enterprise (e.g., SAML-based, ADFS, Ping, Okta).<br/><br/>Go to the dashboard and turn on the methods you want to allow; they will automatically show up in the login/sign-up page. By default, email/password and Google are enabled.
+  1. <h4>Configure the sign-in methods</h4>Auth0 supports a wide range of authentication methods: regular username/password (users can be stored in Auth0 or your own database), social (i.e., Google, Facebook, and 50+ other providers), passwordless (email magic link, email code, and phone code), and enterprise (e.g., SAML-based, ADFS, Ping, Okta).<br/>Go to the dashboard and turn on the methods you want to allow; they will automatically show up in the login/sign-up page. By default, email/password and Google are enabled.
```

Comment:
With only one break between paragraphs, there is no spacing, which looks like a mistake rather than a conscious design choice.

Reply:
I'm unsure if it's documented anywhere, but the common practice in the repo (and in most MD I've seen written) is to never have double blank lines. The linter we were running for awhile would actually stop the build if there were doubles, for what that's worth.

Also, in this case, ordered lists should have no spacing between items at all. If in this case you need separate h4s and content beneath them, I'd recommend that you make the numbering part of the heading (`<h4>3. Use an OpenID Connect SDK to trigger the flow</h4>`)